### ISSUE TITLE
Pretrained Model details

### ISSUE THREAD

**Issue by rajeshh099** [seongmin-kye/meta-SR#4]
Hi,

Could you please give details about the available pretrained model(100th checkpoint).
Using which loss this pretrained model trained from below:-
1. Softmax
2.Prototypical without global classification
3.Prototypical with global classification

Thanks

**Comment by seongmin-kye** [comment]
Hi. We use 3 as our objective.

Thanks.

**Comment by rajeshh099** [comment]
> Hi. We use 3 as our objective.
> 
> Thanks.

Hello,

Thanks for the quick response.
We have also used 3 for our training. But we are getting 2.41 as EER for 100th checkpoint for EERFull.

Is it expected or are we doing something wrong?

Thanks

**Comment by seongmin-kye** [comment]
Results may vary depending on weight initialization.

Or it may not be fully optimized.

**Comment by rajeshh099** [comment]
> Results may vary depending on weight initialization.
> 
> Or it may not be fully optimized.

Hi,

I have just used the code as it is present in GitHub. You mean the EER 2.08 for pretrained model provided by you might be different from GitHub .
Below steps I have performed:-
1.Code base downloaded from github.
2.Data preprocessing and Feature extraction performed as per the instruction given in github page.
3.Trained with option 3.
4.Tested 100th checkpoint for EERFull. We got 2.41.

We were expecting Similar EER of 2.08 as it was mentioned pretrained model was also 100th checkpoint.
Please suggest if we ae missing anything 

Thanks

**Comment by seongmin-kye** [comment]
No. I mean that the seed used for pretraining may be different. According to seed, weight initialization can be differ.

**Comment by rajeshh099** [comment]
> No. I mean that the seed used for pretraining may be different. According to seed, weight initialization can be differ.

Ok. You mean with the same setup if we adjust the weight of hyper parameter, EER can come down to 2.08?

**Comment by seongmin-kye** [comment]
Yes. You can achieve 2.08 or more less EER with other random seed. Also, More training epoch may be helpful.

**Comment by rajeshh099** [comment]
> Yes. You can achieve 2.08 or more less EER with other random seed. Also, More training epoch may be helpful.

Ok. Thanks for the quick response. Appreciated.
